nextage_moveit_config/test/test_moveit.py

```diff
@@ -290,7 +290,7 @@ def test_botharms_plan(self):
         target_pose_l.orientation.w = q[3]
         botharms.set_pose_target(target_pose_r, 'RARM_JOINT5_Link')
         botharms.set_pose_target(target_pose_l, 'LARM_JOINT5_Link')
-        self.assertTrue(botharms.go())
+        self.assertTrue(botharms.go() or botharms.go() or botharms.go())
```

Comment:
Does this mean that the success rate of `botharms.go()` with this particular condition is 1/3?